main.js

```diff
@@ -0,0 +1,34 @@
+const menu = document.querySelector('.nav-links');
+const container = document.querySelector('.nav_container');
+const menuBtn = document.querySelector('#open-menu-btn');
+const closeBtn = document.querySelector('#close-menu-btn');
+const list = document.querySelectorAll('.item');
+
```

Comment:
- [ ] You are almost there! However, as the previous reviewer said, there is a bug when you open/close the menu twice:

![Captura de pantalla 2022-06-13 a la(s) 7 52 34 p  m](https://user-images.githubusercontent.com/83717285/173471934-02c563eb-9f14-43af-9ca0-186cd0b71b83.png)
Take a look at the Dev tools:

|Open 1st time |Close 1st time| Open 2nd time |Close 2nd time|
|---|---|---|---|
|![Captura de pantalla 2022-06-13 a la(s) 7 54 52 p  m](https://user-images.githubusercontent.com/83717285/173472347-4153329e-7b56-4c7b-ba8b-68d8622a509f.png)|![Captura de pantalla 2022-06-13 a la(s) 7 55 05 p  m](https://user-images.githubusercontent.com/83717285/173472358-ce502822-192b-46bb-a866-60af7181d6ae.png)|![Captura de pantalla 2022-06-13 a la(s) 7 55 16 p  m](https://user-images.githubusercontent.com/83717285/173472366-25432670-4fc7-4705-834f-4c9245af76e6.png)|![Captura de pantalla 2022-06-13 a la(s) 7 55 25 p  m](https://user-images.githubusercontent.com/83717285/173472373-a37d2bf2-f0e3-4a4e-a1c5-9ae9ece78cb0.png)|

I would suggest checking how is the `toggle` working, you could try with `add` or `remove` classes.

Reply:
@smunozmo but in my browser its working and also it was my last chance for review

Reply:
Hi @fahimy143 

I'm sorry to hear that, I would suggest in that case asking for a second opinion, this way a Senior CR can help you to solve this issue. 🤓 

You can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).